server/service/PublicationService.js

```diff
@@ -36,17 +35,26 @@ export class PublicationService {
    */
   static search(query) {
     let sortParams = getSortParams(query.orderBy);
-    let genreParams = getFindParams(query.filterByGenres, query.user);
-    let textSearchParams = getTextSearchParams(query.textSearch);
-
-    if (genreParams["genres"]) { textSearchParams["genres"] = genreParams["genres"] }
-
-    let findParams = textSearchParams;
+    let findParams = getFindParams(query.filterByGenres, query.user);
```

Comment:
A branch "issue-91-fix" deve ir primeiro(o método search foi atualizado). Dá uma olhada quando for dado o merge(possiveis problemas de merge)

Reply:
Dar fix nos problemas de merge(Manter o que está na develop).

Reply:
Mas isso quem tem que fazer sou eu ou quem vai revisar e aceitar a pull? hmm

Reply:
Acho que quem deve fazer é tu, porque tem coisa de comentário que não sei pra onde vai.

Reply:
O problema é que eu não sei o que você alterou de forma exata. Você precisou mudar a função search de alguma forma?

Reply:
Eu deixei a função search do jeito que estava na primeira implementação antes da #123...

Para a resolução da #123, criei "searchByText(query)"